solid2/README.md

```diff
@@ -0,0 +1,51 @@
+# Solid Hello World
+
+This is a simple application illustrating how to get started with [Solid](https://solidproject.org/).
+
+It only has two dependencies: [an authentication library](https://github.com/inrupt/solid-client-authn-js) and [solid-file-client library](https://github.com/jeff-zucker/solid-file-client). Everything else is plain HTML, CSS and JavaScript. All the functionality related with Solid is contained in a single file; `solid.js`.
+
+## Understanding the code
+
+If you're not familiar with the basics of Solid, we strongly suggest that you check out [the glossary](Glossary.md). You can find more specific documentation about the code in this app within the [solid.js](./solid.js) file.
+
+The `index.html` and `main.js` files are not documented, but they should be fairly easy to understand if you're already familiar with HTML and JavaScript. The application doesn't have any custom CSS because it uses a classless CSS framework called [Simple.css](https://simplecss.org).
+
+## Usage instructions
+
+If you want to play around with the application, you'll need to log into a [Solid POD](https://solidproject.org/users/get-a-pod). If you prefer to run one in your local environment, we suggest that you use the [Community Solid Server (CSS)](https://github.com/solid/community-server) with the filesystem configuration. This will use your filesystem to serve a Solid POD from the folder of your choice; `./solid-pod` in this case:
+
+```sh
+npm install -g @solid/community-server
+community-solid-server -c @css:config/file.json -p 4000 -f ./solid-pod
+```
+
+If you want to modify the code, you'll also need to serve the application in a url. You could just open the `index.html` file in a browser, but unfortunately that will not work because the authentication flow performs a redirect and that won't work with a website being served with the `file://` protocol.
+
+Any tool to run a local server will work, for example you could use [ViteJS](https://vitejs.dev/):
+
+```sh
+npm install -g vite
+vite .
+```
+
+## FAQs
```

Comment:
I think for now we can leave this as is, but at some point we could have them in a single place instead of duplicated.